apps/src/templates/lessonOverview/LessonNavigationDropdown.jsx

```diff
@@ -46,6 +49,20 @@ export default class LessonNavigationDropdown extends Component {
 
   handleDropdownClick = listItem => {
     if (listItem.link) {
+      firehoseClient.putRecord(
+        {
+          study: 'lesson-plan',
+          study_group: this.props.isStudentLessonPlan
+            ? 'student-lesson-plan'
+            : 'teacher-lesson-plan',
+          event: 'navigate-between-lessons',
+          data_json: JSON.stringify({
+            startingLessonId: this.props.lesson.id,
+            endingLessonId: listItem.id
+          })
+        },
+        {includeUserId: true}
+      );
       navigateToHref(linkWithQueryParams(listItem.link));
```

Comment:
I believe this is going to need to go in a callback on `putRecord`, otherwise it's possible that page navigation will interrupt the PUT request

Reply:
Updated! Thank you!